### PR TITLE
Update compile_to_js.sh to explicite export fucntions

### PR DESCRIPTION
**Pull request by marwin1991**
When compile with WASM=0 functions are not exported. I reported it here kripken/emscripten#7721
I think for now we can stay with this explicite function and we can remove unused in WEB to make compiled library lighter.

### CODE REVIEW

File: distributions/JavaScript/compile_to_js.sh [hunk 1]
```diff
@@ -21,8 +21,457 @@ ls .
 emcmake cmake ..
 emmake make
 
+funs='['
```
Comment (grzanka) [inline distributions/JavaScript/compile_to_js.sh]: Do we have a script which generates list of functions, as below ?
If yes, add a comment line mentioning how to do it

Comment (marwin1991) [reply distributions/JavaScript/compile_to_js.sh]: It is described in JS_INSTALL.txt file

**Comment by grzanka** [comment]
Why tests are failing ? On master they run without problems. Is it a "feature" of forjs branch ? Can you adjust travis for this branch ?

File: distributions/JavaScript/compile_to_js.sh (hunk 1 above)
Comment (grzanka) [reply distributions/JavaScript/compile_to_js.sh]: ok

**Comment by marwin1991** [comment]
```
0.09s$ cd distributions/Python/pyamtrack && python generate_Py_wrapper.py
Reading NAMESPACE file
Harvesting function definitions in header files
Traceback (most recent call last):
  File "generate_Py_wrapper.py", line 46, in <module>
    main()
  File "generate_Py_wrapper.py", line 40, in main
    c_functions = py_parser.harvest(desired_func)
  File "/home/travis/build/libamtrack/library/distributions/Python/pyamtrack/py_parser.py", line 250, in harvest
    elif string.split(string.split(tmp_header_content[tmp_definition_start])[1], '(') [0] in functions:              
TypeError: list indices must be integers, not type
The command "cd distributions/Python/pyamtrack && python generate_Py_wrapper.py" exited with 1.
0.05s$ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then LD_LIBRARY_PATH=$HOME/usr/lib python pyamtrack.py; fi
python: can't open file 'pyamtrack.py': [Errno 2] No such file or directory
The command "if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then LD_LIBRARY_PATH=$HOME/usr/lib python pyamtrack.py; fi" exited with 2.
```

Why this is failing? Can you look at it ?

**Comment by grzanka** [comment]
@marwin1991 lack of doxygen comments in include/AT_CernlibFuns.h ?

File: .travis.yml [hunk 2]
```diff
@@ -34,4 +34,6 @@ script:
   - $HOME/usr/bin/amtrack_test
   - $HOME/usr/bin/amtrack_demo
   - cd distributions/Python/pyamtrack && python generate_Py_wrapper.py
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then LD_LIBRARY_PATH=$HOME/usr/lib python pyamtrack.py; fi  # FIX IT TO WORK WITH OSX
\ No newline at end of file
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then LD_LIBRARY_PATH=$HOME/usr/lib python pyamtrack.py; fi  # FIX IT TO WORK WITH OSX
+  # Tigger rebuild image in docker hub libamtrack/libamtrackforjs
+  - curl -X POST https://cloud.docker.com/api/build/v1/source/f382631a-cc27-4630-9c15-79598e025ea0/trigger/65027d52-ffaf-46f4-8006-96614edda072/call/
```
Comment (grzanka) [inline .travis.yml]: Is URL completely transparent ? Shouldn't it be more secured to avoid random people triggering docker hub rebuild ?

Comment (marwin1991) [reply .travis.yml]: Done

**Comment by marwin1991** [comment]
@grzanka Do you think we can merge this to forjs branch?
Forjs branch will be waitng till we will merge Landau and Vavilov C implementation to master and then merge from master to forjs branch and see if everything works

**Comment by grzanka** [comment]
It is OK now, please merge.

**Pull request merged**